pages/1.13/cli/command-reference/dcos-node/dcos-node-list/index.md

```diff
@@ -30,12 +30,21 @@ dcos node list [OPTION]
 
 | Name, shorthand |  Description |
 |---------|-------------|
-| `--field dcos node`   |  Specifies the name of one or more additional fields you want to include in the output of the dcos node command. You can include this option multiple times to add multiple fields. |
+| `--field name`   |  Specifies the name of one or more additional fields you want to include in the output of the dcos node command. You can include this option multiple times to add multiple fields. |
 | `--help, h`   |   Displays usage. |
 | `--info`   |  Displays a short description of this subcommand. |
 | `--json`   |    Displays JSON-formatted data. |
 | `--version`   |  Displays version information. |
 
+# Permissions
+
+To use this command to return public-facing IP addresses, your service or user account must have the following administrative permission:
+<p>
+<code>dcos:adminrouter:ops:networking full</code>
+</p>
```

Comment:
Thanks for documenting these! I was just checking and there are also other permissions which are important:
- `dcos:adminrouter:ops:mesos full`	
- `dcos:adminrouter:ops:mesos-dns full`

If the user misses one of these 2 permissions, the command will return with an error.

Reply:
Thanks for the additional info (in an ideal world, I'd like to include this section for every cmd line program). Adding this tidbit to the source.